doc/theory/glossary.tex

```diff
@@ -235,12 +232,12 @@
 	\subsection*{Vietoris-Rips complex and Vietoris-Rips persistence} \label{vietoris-rips_complex_and_vietoris-rips_persistence}
 	
 	Let $(X, d)$ be a 
-	% \hyperref[finite_metric_spaces_and_point_clouds]{finite metric space}
+	% \hyperref[finite_metric_spaces_and_point_clouds]{finite metric_space}
```

Comment:
Typo: "finite metric_space" should be "finite metric space" (displayed, not label name)

Reply:
Great! Only this one still left...